tests/tests_integration/tests_db_connection_integration/test_postgres_integration.py

```diff
@@ -898,15 +902,20 @@ def table_finalizer():
     table_finalizer()
 
     df = postgres.execute(f"{{call {func}()}}")
+    result_df = pandas.DataFrame([[1]], columns=["result"])
     processing.assert_equal_df(df=df, other_frame=result_df)
     table_finalizer()
 
-    # fetch is read-only
-    with pytest.raises(Exception):
-        postgres.fetch(f"SELECT {func}() AS result")
+    # works with readOnlyMode=transaction, because autoCommit is true:
+    # https://jdbc.postgresql.org/documentation/use/
+    # fails with readOnlyMode=always, but it is not compatible with pbgouncer
```

Comment:
```suggestion
    # fails with readOnlyMode=always, but it is not compatible with pgbouncer
```
four such places, but we understand anyway